packages/grafana-toolkit/src/cli/index.ts

```diff
@@ -176,6 +177,19 @@ export const run = (includeInternalScripts = false) => {
       });
     });
 
+  program
+    .command('plugin:sign')
+    .option('--signatureType <type>', 'Signature Type')
+    .option('--rootUrls <urls...>', 'Root URLs')
+    .description('Create a plugin signature')
```

Comment:
the cli should maybe also accept `GRAFANA_API_KEY` and the help should say something about it

Reply:
Not sure that we want to encourage users passing the api key on the command line, which is why I was following the environment variable pattern from ci-build.  We could support passing the location of a file to read it from maybe, do we have precedent for that anywhere else?

Reply:
i agree with Dan, putting api keys in command line args isnt a great idea.
reading from an ENV var is going to be the most common approach used from CI/CD tooling.  For those building and signing locally, a config file would be good. we could allow the path to the file to be set, with a default of `~/.local/share/grafana-toolkit/config` or similar

A future enhancement could be to support a login command that uses an oauth flow, similar to how Google's `gcloud` or Azure's `az` commands work.
ie, users run
`grafana-toolkit --login`, which then opens grafana.com in a browser, authenticates the user and then passes back a token to grafana-toolkit that it can persist in the config file.